consensus/istanbul/backend/backend.go

```diff
@@ -210,6 +210,7 @@ type Backend struct {
 
 	aWallets atomic.Value
 
+	wg           sync.WaitGroup
```

Comment:
Given the amount of properties backend already has, I'd consider adding a much more descriptive variable name for this waitgroup (and a small comment explaining why it may be needed). `closeReadyWG`, or something along those lines

Reply:
So it looks like we may just be able to remove this wg, since the go-routine it was waiting for doesn't appear to be necessary.